### ISSUE TITLE
1.4.1 upgrade with error message

### ISSUE THREAD

**Issue by antoineverger** [TYPO3/CmsComposerInstallers#67]
Hi dear commiters of CmsComposerInstallers,

when upgrading your library on my installation on MacOS, I got the following error message : 

 ``` 
- Updating typo3/cms-composer-installers (1.3.1 => 1.4.1): Downloading (100%)
Writing lock file
Generating autoload files


  [Symfony\Component\Debug\Exception\FatalThrowableError]
  Class 'TYPO3\CMS\Composer\Plugin\Core\AutoloadConnector' not found
```

I can provide more infos if required !

Best regards,

Antoine

**Comment by thommyhh** [comment]
I have experience the same problem. I think it might be, that the class just has been removed. On my MacOS with the default PHP version, 1.4.* does not work anyway, because it needs PHP 7.

Which version do you have?

**Comment by mbrodala** [comment]
I'd assume it's the same issue as #55 back then. There is no real solution aside from running the same command again.

**Comment by helhum** [comment]
Yes, same issue. sorry for that. I'll try to mitigate that in 1.4.2

**Comment by helhum** [comment]
> On my MacOS with the default PHP version, 1.4.* does not work anyway, because it needs PHP 7.

composer does not allow you to upgrade to this version, if your PHP version does not match.

You can add https://getcomposer.org/doc/06-config.md#platform to your composer.json, if you want to enforce a certain PHP version for your project

**Comment by thommyhh** [comment]
I need to use "--ignore-platform-reqs" to install TYPO3 CMS 8 locally on my mac for usage in the IDE. So this check is unfortunately disabled. I know, that this is my own problem. ;-)

**Comment by helhum** [comment]
You can also enforce the older version of the installers by doing `composer req typo3/cms-composer-installers ~1.3.0` for your package

**Comment by thommyhh** [comment]
Thank you. That's exactly what I did. So this was the right choice, except upgrading PHP on my mac and having the same problem with older projects the other way round. ;-)

**Comment by antoineverger** [comment]
Thanks guys, I enforced a lower version and it´s OK now. @helhum do you want me to close the issue or keep it open as a reminder ?

**Comment by helhum** [comment]
With released version 1.4.2, upgrade from 1.3.x should be smooth (PHP 7 requirement aside of course)
